website/static/css/pages/remove-contributor-page.css

```diff
@@ -0,0 +1,4 @@
+.col-md-4 {
+    padding-top: 20px;
+    padding-right: 15px;
+}
```

Comment:
We are not using RequireJS; we use Node-style/CommonJS imports. Webpack parses the `require` statements and does the transforms/bundling/minification. We use the `css-loader`, which allows us to import CSS modules.